artwork.json

```diff
@@ -41,6 +55,20 @@
     },
     "license": "MIT"
   },
+  {
+    "date": "2020-05-14",
+    "image": "/images/deno_city.jpeg",
+    "title": "Deno in the City",
+    "alt": "the deno mascot standing between a some tall lit up buildings at night",
+    "artist": {
+      "name": "Dimitrij Agal",
+      "profile_image": "https://pbs.twimg.com/profile_images/1242014548896264194/VJJtVXVG_400x400.jpg",
+      "twitter": "masbagal",
+      "github": "masbagal",
+      "web": "https://masbagal.com/"
+    },
+    "license": "MIT"
+  },
```

Comment:
Should default to restricted license. @masbagel can come back and change it - or ok it here

Reply:
I'm okay with MIT